src/shared/modules/dbMeta/dbMetaDuck.ts

```diff
@@ -697,6 +707,7 @@ export const serverConfigEpic = (some$: any, store: any) =>
           return Rx.Observable.of(null)
         })
     })
+    .do(() => store.dispatch(update({ serverConfigDone: true })))
```

Comment:
Not sure I'm a big fan of using this as a one-time thing - because this epic runs at an interval and the value can change between runs. 
Using it like this kind of implies that it only runs once.
A name change might be good enough. `serverConfigReads: Int` maybe to really emphasize that it's a recurring thing?

I'm going to approve this anyway now, but give it a think and change if you want.

Reply:
This new state opens up for browser to be in a pending state for other areas as well (all other configs just above this) rather than having a local default to fallback on.

Reply:
I think serverConfigReads is a better implementation, good idea!